src/style.scss

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+
+  @for $row from 1 through 5 {
+    &--#{$row}:nth-child(-n + #{$row}) {
+      background-image: url(images/star-active.svg);
+      background-position: 0;
+    }
+  }
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-size: cover;
+  background-position: center;
+  height: 12px;
+  width: 12px;
```

Comment:
Star size is 16x16 due to figma
![image](https://github.com/mate-academy/layout_stars/assets/80700247/c7a3a210-e8e0-4b46-9660-ac8291b92320)